{
  "id": "MAL-2024-5893",
  "summary": "Malicious code in requestlib (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "aliases": [
    "SNYK-PYTHON-REQUESTLIB-5932264"
  ],
  "modified": "2024-06-28T03:15:13.720226Z",
  "published": "2024-06-25T13:41:39Z",
  "database_specific": {
    "malicious-packages-origins": [
      {
        "modified_time": "2024-06-25T13:41:39Z",
        "import_time": "2024-06-28T02:50:30.573587881Z",
        "versions": [
          "1.0.2",
          "1.0.0"
        ],
        "id": "RLMA-2024-04694",
        "source": "reversing-labs",
        "sha256": "ab7c83c55c8537bdb162c03a90e627b47521b27d4c142f064f1a76148bf0e367"
      }
    ]
  },
  "references": [
    {
      "type": "ARTICLE",
      "url": "https://medium.com/checkmarx-security/the-evolutionary-tale-of-a-persistent-python-threat-d19196d4ed15"
    },
    {
      "type": "ADVISORY",
      "url": "https://security.snyk.io/vuln/SNYK-PYTHON-REQUESTLIB-5932264"
    },
    {
      "type": "ADVISORY",
      "url": "https://research.jfrog.com/malicious-packages/"
    }
  ],
  "affected": [
    {
      "package": {
        "name": "requestlib",
        "ecosystem": "PyPI",
        "purl": "pkg:pypi/requestlib"
      },
      "versions": [
        "1.0.2",
        "1.0.0"
      ],
      "database_specific": {
        "source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/requestlib/MAL-2024-5893.json"
      }
    }
  ],
  "schema_version": "1.6.0",
  "credits": [
    {
      "name": "ReversingLabs",
      "contact": [
        "https://www.reversinglabs.com"
      ],
      "type": "FINDER"
    }
  ]
}